doc/processes/release_process.md

```diff
@@ -0,0 +1,109 @@
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Eclipse OpenJ9 Release Process
+
+The OpenJ9 JVM is composed of two code repositories:
+
+* OpenJ9 [https://github.com/eclipse/openj9]()
+* OMR [https://github.com/eclipse/openj9-omr]()
+
+OpenJ9 needs to be combined with code from OpenJDK to create a Java SDK.
+The OpenJDK code is managed in separate github repos referred to as the 
+"OpenJDK extensions for OpenJ9 for JDK X".  Currently there are two 
+extensions repos:
+
+* JDK8 [https://github.com/ibmruntimes/openj9-openjdk-jdk8]()
+* JDK9 [https://github.com/ibmruntimes/openj9-openjdk-jdk9]()
+
+OpenJ9 binaries are built by the AdoptOpenJDK community.  
+
+An OpenJ9 release should:
+
+* Use a single code base to support the various JDK-levels.
+* Track the upstream OpenJDK releases.  
+* Use a consistent set of tags across OpenJ9 & extensions repos to 
+ensure that releases can be rebuilt or branched later.
+* OpenJ9 releases should be high quality and pass a common quality bar. 
+* Communicate the tags for a release to AdoptOpenJDK to ensure that
+binaries at the correct levels have been created. 
+* Clearly define the supported platforms
+* Should not regress performance from release to release.
+
+
+## Release cadence
+OpenJ9 should aim to release quarterly to match the OpenJDK long term 
+support (LTS) release cadence.  OpenJDK 8 is released quarterly and 
+other LTS releases will likely follow similar release cadences.  There 
+should be monthly milestone builds as the project builds towards the 
+next release.
+
+## Quality bar
+Work with the AdoptOpenJDK community to define a common quality bar for 
+any OpenJDK-based JDK.  The aim is to define a set of tests, both 
+functional and stress, and 3rd party applications that should pass for 
+every release.
+
+## Platform support
+Define the platforms that are supported by this release.  The set of 
+platforms will depend on the availability of hardware at OpenJ9 and 
+AdoptOpenJDK that can be used to validate the platform meets the quality 
+bar.  The code base may support more platforms than can be validated by 
+the quality.  Such platforms will not be officially included in the release.
+
+## Performance
+OpenJ9 should avoid regressing performance from release to release.
+This is harder to validate in the open project given the limitted 
+hardware resources available.  Work is being done at AdoptOpenJDK to 
+enable performance testing.
+
+# Release Process
```

Comment:
I sent an email about this to the mailing list but only one response by @mstoodle was recieved. Quoting it here for visibility:

> Has anyone proposed the idea of having "GitHub releases" (https://github.com/eclipse/openj9/releases) which correspond to official Eclipse releases? i.e. I would love to have a technical overview of exactly what went into a release. From a developers point of view the things developers may care about is:
> 
> - Release Notes
> - Notable Issues fixed
> - What commit of all the repositories that go into Eclipse OpenJ9 a particular release is based off
> 	- For example if I am looking for whether a particular Pull Request for an Issue fix that is relevant to me is in a particular release I would need to know the commit SHAs
> 
> A very good example of this can be seen on the .NET Core project:
> https://github.com/dotnet/coreclr/releases
> 
> Each release is tagged to a particular git commit SHA (you can click on it on the left). It has versioning numbers. GitHub conveniently diffs the git commits between this version and the last automatically. Each commit corresponding to a version is git tagged (https://github.com/dotnet/coreclr/tags). All tags have release notes which can point to a Markdown document right inside the repository (https://github.com/dotnet/core/blob/master/release-notes/2.0/2.0.3.md). These Markdown documents detail all release notes including linking to the official announcement and release (this can be the Eclipse release). It lists important information such as security considerations, notable issues fixed, etc. 
> 
> This can also be a great place to list Acknowledgements to all the contributors in this release so it recognizes the developers working on OpenJ9 giving contributors another thing to be proud about. And of course we can link directly to the AdoptOpenJDK build which corresponds to this particular release, as well as docker images, etc.
> 
> In essence the GitHub release would have more technical information and link to the official Eclipse release which can contain more highlevel content non-developers may care about.

Since we are already hosted on GitHub why not take advantage of features it provides for release management out of the box as users will expect to see such information on the repository releases page (https://github.com/eclipse/openj9/releases).

Reply:
I think this is a great idea.